frontend/website/src/Testnet.re

```diff
@@ -211,6 +293,94 @@ module Styles = {
         ),
       ]),
     ]);
+
+  let buttonRow =
+    style([
+      display(`flex),
+      flexWrap(`wrap),
+      marginLeft(`auto),
+      marginRight(`auto),
+      marginTop(rem(3.)),
+      marginBottom(rem(3.)),
+      justifyContent(`spaceAround),
+      alignItems(`center),
+      flexDirection(`column),
+      media(Style.MediaQuery.notMobile, [flexDirection(`row)]),
+    ]);
+
+  let ctaButton =
+    style([
+      padding(`rem(1.125)),
+      background(`rgba((71, 137, 196, 0.1))),
```

Comment:
is this a color we intend to continue using? let's add it to the Colors module if so

Reply:
was a color already in Style.re- changed to teal